vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,10 +1,179 @@
+
+* Most of the existing ECG (Blueprint) vulnerabilities have been fixed; regardless, the user is strongly advised to rerun vulnerability scans every time a new bundle is generated. The remaining vulnerabilities will be fixed in the next minor version of Entando. These are not exploitable in production and only exposed during the internal development process. In particular, they concern:
+
+    * The dev webserver, used when developing the app
+    * The Jest test files, used when running tests with jest
+    * General webpack configuration
+    * Dev utilities, used in the dev environment
+    * The PostCSS utility, used to import the CSS when building the final artifact
 
 ## New Features and Major Fixes
 
-### Entando Application
+### Entando Hub
+
+The [Entando Hub](../getting-started/landing-page.md#entando-hub) is a new [Entando Component Repository](../../docs/getting-started/concepts-overview.md#entando-component-repository) feature that allows different EntandoApps to share bundles. The [Entando App Builder](../../docs/getting-started/concepts-overview.md#entando-app-builder) now includes a feature that allows curators to [browse, deploy, install and upgrade bundles from one or more Entando Hub instances](../../tutorials/solution/entando-hub.md#automatically-access-the-hub-from-the-app-builder). 
+### Application Engine
+
+* Entando has implemented the SQL versioning tool Liquibase to track, manage and deploy updates to databases. Refer to [Liquibase Migration](../reference/databases.md#liquibase-migration) to learn how the App Engine automates database schema management and modifications.
+* The latest MySql 8.x and PostgreSQL 14 versions are supported.
+* Oracle 18c as an external DBMS is supported.
+* Infinispan is now used only for EAP session sharing. 
+* [Object cache](../../tutorials/devops/caching-and-clustering.md) is implemented in Redis.
+* Logging infrastructure has been moved to Logback.
+* CMS content API performance optimizations have been implemented.
+
+### App Builder
+
+* Added dash (-) support for fragment code validation.
+* Improved UI performance affected by instances containing significant content and assets.
+* Implemented performance improvements in page management.
+* Resolved several extended bugs hindering performance and user experience.
+* Reduced the size of the application state and started phasing out obsolete technologies.
+
+### Entando Component Generator (ECG)
+
+* Upgraded JHipster to version 7.2.
+* ECG now allows the use of a BOM curated by Entando and containing more recent dependencies than is standard for a specific version of JHipster.
+* Removed the .map files generated from Blueprint-generated code.
+* Implemented bug fixes and error handling improvement.
+* Switched to bundle plugin descriptor version 4.
```

Comment:
Let's add a sentence - "This changes the default ingressPath to not include the image version and adds support for environment variables."